### PR TITLE
alpha value upgradation

### PR DESCRIPTION
**Pull request by RishikeshB**
Hi,
Thank you @kurkle for this great library! As in issue **#93** I'd like to increase chart color intensity for better visualization. For this added '**alpha**' value where set alpha value to 1 (**alpha:1**) the intensity of the chart flow color increases or else by default the color value will be 0.5. This will be helpful when given only two color for large dataset.

Eg: I'm not given alpha value and not given any color for flow, chart takes its default color values has red and green :
code:
```
data: {
datasets: [{
label: 'My sankey',
data,
}]
},
```
**chart looks like:**
alpha=0

Eg: I'm given alpha value as 1 and not given any color for flow, chart takes its default color values has red and green:
code:
```
data: {
datasets: [{
label: 'My sankey',
data,
alpha:1
}]
},
```
**chart looks like**
alpha=1

I'm submitting a PR that proposes a new option.

### CODE REVIEW

**Comment by stockiNail** [comment]
@RishikeshB @kurkle if I may jump in the thread, maybe the best thing is to remove any default alpha color value and delegate the user to decide the colors to apply. Just thinking loud.

**Comment by kurkle** [comment]
I agree with @stockiNail, it would be best to remove the .alpha call and use whatever already is in the color provided.

**Comment by RishikeshB** [comment]
but when we remove .alpha call, any of us may required a lite colors to view flow. alpha value will serve better usage switching from dark and light.

File: .gitignore [hunk 1]
```diff
@@ -3,6 +3,5 @@
 bower.json
 cc-test-reporter
 coverage/
-dist/
```
Comment (kurkle) [inline .gitignore:6]: Its not planned to host the bundled files in GH, so this change is not acceptable.

Comment (RishikeshB) [reply .gitignore]: added -dist/ to gitignore.

Comment (RishikeshB) [reply .gitignore]: image

PR request failed due to the above error

**Comment by kurkle** [comment]
please remove the added files in `dist` too

**Comment by kurkle** [comment]
Also this should be documented, can you do that too?

**Review by kurkle — APPROVED** [review]

**Pull request merged**